docs/guides/migrations.md

```diff
@@ -0,0 +1,83 @@
+
+```javascript
+keystone.createList('MigrationExample', {
+  fields: {
+    age: { type: Select, options: '0-18, 19-35, 36-50, 50+' },
+  },
+});
+```
+
+If we restart Keystone after this change it will throw an error. The `data_type` of the `age` column is now a `text` value and it should contain a string representation of the age range. The table should be change to something like the following:
+
+| table_name  | column_name | data_type |
+| ----------- | ----------- | --------- |
+| ExampleList | id          | integer   |
+| ExampleList | age         | text      |
+
+Keystone cannot make this change for you because there may be existing data with a type of `integer`. Sometimes migration is as simple as converting an `integer` to a `string` but in certian cases, such as this, it requires programatically resolving the appropriate.
+
+## Migration options
+
+In development sometimes it can be helpful to drop the database everytime Keystone is restarted. To do this using the `KnexAdapter` you can pass the `dropDatabase` option to the adapter:
+
+```js
+const { KnexAdapter } = require('@keystone-alpha/adapter-knex');
+const adapter = new KnexAdapter({ dropDatabase: true });
+```
+
+This might be enough for small projects but it won't be ideal for everyone. If you want data to persist in development you need to re-initialise the database or preform migrations. And regardless of the development environment, almost all projects will eventually need to make changes to a List table in production. For this there are a number of solutions.
+
+The most basic solution is to make these changes on the database. You can do this via a command-line, but if you're reading this guide you're probably looking for an application to help you view and manage tables created by Keystone. For Mongo you might try (Studio 3T)[https://studio3t.com/download/] which is free for non-commercial use. For Postgres you can try (pgadmin)[https://www.pgadmin.org/]. There are a lot of other options and you should find one that works for you.
+
+Once you get familiar with your database GUI you will be able to look at the tables Keystone creates for you, manually drop them, make change to Lists and see how the data-structure differs, and eventually write migration queries such as:
+
+```sql
+ALTER TABLE "ExampleList"
+ALTER COLUMN "age" TYPE text
+    USING (
+    CASE WHEN age<=18 THEN '0-18'
+        WHEN age<=35 THEN '19-35'
+        WHEN age<=50 THEN '36-50'
+        ELSE '50+'
+    END);
+```
+
+Keystone database adapters use Mongoose and Knex under the hood, both of which have migrations solutions. Rather than running SQL queries directly, you can write migrations using JavaScript, and tools like the (Knex CLI)[http://knexjs.org/#Migrations] allow you to keep track of which migrations have been applied. You seed the database, run or rollback specific migrations or apply all missing migrations in the correct order.
```

Comment:
(Link fix)
```suggestion
Keystone database adapters use Mongoose and Knex under the hood, both of which have migrations solutions. Rather than running SQL queries directly, you can write migrations using JavaScript, and tools like the [Knex CLI](http://knexjs.org/#Migrations) allow you to keep track of which migrations have been applied. You seed the database, run or rollback specific migrations or apply all missing migrations in the correct order.
```